articles/models.py

```diff
@@ -0,0 +1,59 @@
+import uuid
+from operator import attrgetter
+from typing import Any
+
+from autoslug import AutoSlugField
```

Comment:
Django has its own inbuilt slugify method, use it rather than adding another 3rd party package

Reply:
okay